graphql/resolvers.ts

```diff
@@ -1,23 +1,39 @@
+import { Request } from 'express'
 import { login, logout } from '../helpers/controllers/authController'
 import db from '../helpers/dbload'
+import _ from 'lodash'
 
-const { Lesson, User } = db
+const { User, Submission, UserLesson, Lesson } = db
 
 export default {
   Query: {
     lessons() {
       return Lesson.findAll({
-        include: [
-          'challenges',
-          {
-            model: User,
-            through: {
-              attributes: ['isPassed', 'isTeaching', 'isEnrolled']
-            }
-          }
-        ],
+        include: ['challenges'],
         order: [['order', 'ASC']]
       })
+    },
+    async session(_parent: void, _args: void, context: { req: Request }) {
+      const userId = _.get(context, 'req.session.userId', false)
+
+      if (!userId) {
+        return null
+      }
+
+      const user = await User.findOne({
+        where: { id: userId },
+        include: [Submission, UserLesson]
```

Comment:
include: `Lesson` not `UserLesson`

Reply:
But that wouldn't get me the lesson status information for that user, would it?